core/station/impl/src/models/request_policy_rule.rs

```diff
@@ -44,6 +45,22 @@ pub enum RequestPolicyRule {
     NamedRule(NamedRuleId),
 }
 
+impl fmt::Display for RequestPolicyRule {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            RequestPolicyRule::AutoApproved => write!(f, "AutoApproved"),
+            RequestPolicyRule::QuorumPercentage(_, _) => write!(f, "QuorumPercentage"),
+            RequestPolicyRule::Quorum(_, _) => write!(f, "Quorum"),
+            RequestPolicyRule::AllowListedByMetadata(_) => write!(f, "AllowListedByMetadata"),
+            RequestPolicyRule::AllowListed => write!(f, "AllowListed"),
+            RequestPolicyRule::Or(_) => write!(f, "Or"),
+            RequestPolicyRule::And(_) => write!(f, "And"),
+            RequestPolicyRule::Not(_) => write!(f, "Not"),
+            RequestPolicyRule::NamedRule(_) => write!(f, "NamedRule"),
```

Comment:
Do we really want to hide the nested rules here?

Reply:
Maybe just the structure without details in the leaves?

Reply:
Sounds good

Reply:
Addressed in [e766793](https://github.com/dfinity/orbit/pull/570/commits/e7667933f66cad1d4d6ec6f1b61d54f2f204cf32), I introduced a new field in the error so that it can hold both the offending rule and the full rule.